src/index.js

```diff
@@ -1,20 +1,54 @@
 import path from 'path';
 
-export default function({types: t}) {
+export default function({ types: t }) {
+  function getValue(src, name, key) {
+    const basePath = path.resolve(src);
+    const pkg = require(path.join(basePath, '..', name));
+
+    return pkg[key];
+  }
+
+  function isManifest(node) {
```

Comment:
npm never refers to package.json as a manifest, and that word has overloaded meanings for things like Heroku or Android. Could we just call this `isPackageJSON`?